### PR TITLE
Winget Sandboxie-Classic

### PR DESCRIPTION
**Pull request by okrc**
[vedantmgoyal2009/winget-releaser](https://github.com/vedantmgoyal2009/winget-releaser) `version` input is now supported.

The version number obtained from the `Release title` is used here.

### CODE REVIEW

**Comment by isaak654** [comment]
@vedantmgoyal2009 Is it ok for you?

**Comment by okrc** [comment]
> Is it ok for you?

Can you tell me more about it? My English is terrible and I don't understand what you mean.

**Comment by isaak654** [comment]
I'm asking a review from the GitHub user who originally created the workflow in this repo.

**Comment by isaak654** [comment]
I'm requesting a review from the GitHub user who originally created the workflow in this repo.

**Comment by okrc** [comment]
Only minimal changes were made here for Sandboxie-Classic. Only version input has been added, except for the overlapping parts in Sandboxie-Plus.
Get the version number by intercepting the Release title, for example
```
> "Release v1.3.2 / 5.58.2" -replace '^.*/ '
5.58.2
```
If the format of the header has not changed, it works fine here.
If there is nothing wrong with `vedantmgoyal2009/winget-releaser` itself, this repo will work fine.

File: .github/workflows/winget.yml [hunk 1]
```diff
@@ -6,8 +6,21 @@ jobs:
...
+        uses: vedantmgoyal2009/winget-releaser@latest
         with:
           identifier: Sandboxie.Plus
           installers-regex: 'Sandboxie-Plus.*.exe$'
           token: ${{ secrets.WINGET_TOKEN }}
+      - name: Get Sandboxie-Classic version
+        id: get_version
+        run: |
+          $VERSION="${{ github.event.release.name }}" -replace '^.*/ '
+          Write-Host ::set-output name=CLASSIC_VER::$VERSION
```
Comment (vedantmgoyal2009) [inline .github/workflows/winget.yml:19]:
```suggestion
        run: |
          $VERSION="${{ github.event.release.name }}" -replace '^.*/ '
          Write-Host "::set-output name=CLASSIC_VER::$VERSION"
        shell: pwsh
```

**Review by vedantmgoyal2009 — CHANGES_REQUESTED** [review]

**Review by vedantmgoyal2009 — APPROVED** [review]
Rest everything seems good to me :shipit:

**Pull request merged**